client/src/pages/Login.tsx

```diff
@@ -1,19 +1,55 @@
 import React from "react";
-import { login } from "../api/auth";
-import { useMutation } from "@tanstack/react-query";
+// import { login } from "../api/auth";
+// import { useMutation } from "@tanstack/react-query";
 import { Link, useNavigate } from "react-router-dom";
+import { useEffect } from "react";
+import { API_URL } from "../api/config";
 
 export const Login = () => {
-  const loginMutation = useMutation({ mutationFn: login });
   const navigate = useNavigate();
 
+  useEffect(() => {
+    console.log("Entering login page");
```

Comment:
this logic can be centralized in the root

Reply:
will change this now, thx

Reply:
Is it ok if I ask you questions abt the root later in the lab? I changed it and apparently it is not working as it should... 😢 
<img width="704" alt="Screenshot 2024-04-24 at 12 18 41 PM" src="https://github.com/faizahsayyid/csc490-interactive-art/assets/128764986/7bb4f679-04bf-4a81-98a8-efb4a68eb1a7">